boards/arm/nucleo_h745zi_q/support/openocd.cfg

```diff
@@ -0,0 +1,12 @@
+source [find board/st_nucleo_h745zi.cfg]
+
+$_CHIPNAME.cpu0 configure -event gdb-attach {
```

Comment:
Will this config work for both cores?

Reply:
Really I do not know. It was already placed in the DevVersion of OpenOCD I needed to install. Some suggestions on possible changes? Or should I place the .cfg file here?

Reply:
added support for cpu1.

Reply:
leave only cpu0

Reply:
Tested for both cores separately. The config works for M7 and M4. Simultaneous debugging is still not possible, since OpenOCD does not supports it.